src/test/mediapipe/summator_graph.pbtxt

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+input_stream: "in1"
+input_stream: "in2"
+output_stream: "out"
+node {
+  calculator: "OpenVINOModelServerSessionCalculator"
+  output_side_packet: "SESSION:session"
+  node_options: {
+    [type.googleapis.com/mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
+      servable_name: "add"  # servable name inside OVMS
+      servable_version: "1"
+    }
+  }
+}
+node {
+  calculator: "OpenVINOInferenceCalculator"
+  input_side_packet: "SESSION:session"
+  input_stream: "INPUT1:in1"
+  input_stream: "INPUT2:in2"
+  output_stream: "SUM:out"
+  node_options: {
+        [type.googleapis.com/mediapipe.OpenVINOInferenceCalculatorOptions]: {
+          tag_to_input_tensor_names {
+            key: "INPUT1"
```

Comment:
Since the crash was linked to metrics, maybe we should also test inference with metrics enabled?